spacy/util.py

```diff
@@ -294,7 +294,7 @@ def find_matching_language(lang: str) -> Optional[str]:
 
     # Find out which language modules we have
     possible_languages = []
-    for modinfo in pkgutil.iter_modules(spacy.lang.__path__):  # type: ignore
+    for modinfo in pkgutil.iter_modules(spacy.lang.__path__): # type: ignore[attr-defined]
```

Comment:
Extremely picky detail: but what's reformatting the whitespace in cases like this? Can you check the formatting with the current version of black?

Reply:
Thanks, you're right, somehow the version checked in in the PR seems not to have had the correct formatting. It should be fixed now.